packages/boa/tests/base/basic.js

```diff
@@ -1,52 +1,48 @@
-const test = require('tape');
+const test = require('ava');
 const boa = require('../../');
 const builtins = boa.builtins();
 const { PyGetAttrSymbol, PySetAttrSymbol, PyGetItemSymbol, PySetItemSymbol } = boa.symbols;
 
 test('keyword arguments throws', t => {
-  t.throws(() => boa.kwargs(false), TypeError);
-  t.throws(() => boa.kwargs(true), TypeError);
-  t.throws(() => boa.kwargs('foobar'), TypeError);
-  t.throws(() => boa.kwargs(123), TypeError);
-  t.end();
+  t.throws(() => boa.kwargs(false), { instanceOf: TypeError });
+  t.throws(() => boa.kwargs(true), { instanceOf: TypeError });
+  t.throws(() => boa.kwargs('foobar'), { instanceOf: TypeError });
+  t.throws(() => boa.kwargs(123), { instanceOf: TypeError });
 });
 
 test('hash function', t => {
-  t.ok(builtins.__hash__());
-  t.equal(builtins['__notexists__'], undefined);
+  t.is(Boolean(builtins.__hash__()), true);
```

Comment:
```suggestion
  t.true(Boolean(builtins.__hash__()));
```

Reply:
true is the keyword, I'd prefer `is` instead of the `true` method.